ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -54,6 +55,58 @@ def test_torch_instance_add(
     )
 
 
+# new_ones
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    shape=helpers.get_shape(
+        allow_none=False,
+        min_num_dims=1,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=10,
+    ),
+    dtypes=_dtypes(),
```

Comment:
I reused the st.composite functions _dtypes and _requires_grad from test_creation_ops.py as the test is very similar to test_creation_ops.test_torch_ones().

Reply:
Squashing your commits, it's too much

Reply:
Sure! We squash all PRs before merging anyway :)